react/StoreLink.tsx

```diff
@@ -97,6 +99,7 @@ function StoreLink(props: Props) {
       target={target}
       className={rootClasses}
       scrollOptions={scrollOptions}
+      {...(rel ? {rel: `${rel}`} : {})}
```

Comment:
Same here. Why pass it like this?

Reply:
Hi! I added it like that because a simple rel={rel} would give me this error 
![immagine](https://user-images.githubusercontent.com/35635227/124572615-10c11400-de49-11eb-8a70-6500ef3eee18.png)
another workaround to this could be like this:
![immagine](https://user-images.githubusercontent.com/35635227/124572786-3817e100-de49-11eb-914a-768e0454e98d.png)

Reply:
Thank you for your answer! Does it mean that I should also make a PR for the render runtime?

Reply:
@igorbrasileiro ?

Reply:
I transfer this question to @victorhmp that has more context about it.

Reply:
I think you should make a PR for render-runtime, just adding the type. From what I've seen, you'll only need to update the type @igorbrasileiro linked above and you're good to go.

Reply:
Does it mean that I need to extend the interface of the Props from both types NavigatingOptions and HTMLAnchorElements?

Reply:
From what I understood, it isn't possible for an interface to extend two types simultaneously
![immagine](https://user-images.githubusercontent.com/35635227/125635581-ff20d679-f384-4587-9ea9-f7b8732f7170.png)

Reply:
(I just tried without thinking too much just to check, maybe there's a workaround I'm not aware of)

Reply:
It's not ideal, but I think you could just do it like:

```tsx
interface Props extends NavigateOptions {
   rel: HTMLAnchorElement['rel']
   ...
}
```

Reply:
Sorry for the delay here! But I see that the PR at render-runtime solves this, awesome! :D